app/generate.js

```diff
@@ -1,5 +1,115 @@
+import { template } from 'lodash';
 import { getData } from './data';
+import fs from 'fs';
+import { deleteFolderRecursive } from "./utils";
+
+function createDistFolder(name) {
+  const distName = './dist/' + name;
+  if (fs.existsSync(distName)) {
+    deleteFolderRecursive(distName);
+  }
+  fs.mkdirSync(distName, {recursive: true});
+}
+
+function generateSections(theme, sections) {
+  let retArray = [];
+  const sectionTemplate = template(fs.readFileSync('./templates/' + theme + '/section.tpl.html', 'utf8'));
+  const imageTemplate = template(fs.readFileSync('./templates/' + theme + '/image.tpl.html', 'utf8'));
+  
+  for (let i = 0; i < sections.length; i++) {
+    let section = sections[i];
+
+    if (section.image) {
+      section.imageHtml = imageTemplate(section.image);
+    } else {
+      section.imageHtml = "";
+    }
+    retArray.push(sectionTemplate(section));
+  }
+
+  return retArray;
+    //Does the same thing with header and footer, but with sections.
+    //Puts all the information in an array and saves it in sectionOutput'i'.
+
+}
+
+function generatePage(pages, header, footer, name, sectionOutput) {
```

Comment:
The function is named "generatePage" but the function generates many pages - so add a plural `s`
Also, this page both generates and writes the pages, whereas other `generateSections` only generates the sections.

I want you to return the array with pages from `generatePages` and then have another function that loops through the generated pages and writes them (should probably be named `writePages`).